README.md

```diff
@@ -21,17 +21,35 @@ This library uses the ORS API for request validation. To understand the input of
 
 ## Developement Setup
 
-Clone the openrouteservice-js package from GitHub into a developement environment of your choice.
+### WSL Setup
 
-Make sure you have installed the npm, node.js and nvm package managers to ensure you will be able to work in the repository.
+Download and install an Ubuntu terminal environment.
+Set it as the default distribution in the command line ``wsl.exe --set-default [Distro]``
+You can use ``wsl.exe -l`` to check which distribution is currently the default one.
+
+You might need to update the apt package manager to access and be able to install packages. You can do this with the command ```sudo apt update && sudo apt upgrade```
+
+*Recommended steps (but not required):*
+Install nano and GitHub credential manager.
+
```

Comment:
You can leave this block out. That is not specific to the package.

In general if you want to use the "code notation" you can either use [inline code segments or code blocks](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).
You can also take a look at the other formatting stuff :) you will use them quite a lot